### DISCUSSION TITLE
Can't disable copilot in vscode

### DISCUSSION THREAD

**Discussion by Butanium** [community/community#45131] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

<img width="707" alt="image" src="https://user-images.githubusercontent.com/55806347/214012998-8997352d-d98c-4678-9031-b12e79d033de.png">

When I click "disable copilot" or use my keybinding it stays activated. Uninstall / reinstall doesn't fix the issue. 
VScode version : 1.74.3

**Comment by Butanium** [comment, marked as answer]
Actually, the notification was just hidden because I silenced notifications...

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Butanium ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
